runtime/lua/vim/treesitter/query.lua

```diff
@@ -267,6 +267,50 @@ function M.get_node_text(node, source, opts)
   end
 end
 
+--- Gets the text corresponding to a given query capture
```

Comment:
Do we need a new (public) function for that? Can't you just use `vim.treesitter.query.get_node_text()` to get the text from the `capture_node`?

Reply:
`get_capture_text` also reads `text` and `range` from the metadata

Reply:
what for?

Reply:
what I'm getting at: can't `get_node_text` be extended to do the job?

Reply:
this should also be possible by adding `id` and `metadata` to `opts`, should I do that instead?

Reply:
that's where this was going, yes ;) (If it makes sense; I just want to make sure we _need_ a separate public(!) function -- we already have too many of those...)

Reply:
instead of passing both `id` and `metadata`, I opted to pass `metadata[id]` to `metadata` to simplify the logic a bit. The naming might be a little confusing but I couldn't come up with a different name